rdmo/projects/static/projects/js/project_questions/services.js

```diff
@@ -808,7 +808,13 @@ angular.module('project_questions')
                 if (angular.isDefined(questionset)) {
                     service.initView(questionset.id);
                 } else if (angular.isDefined(section)) {
-                    service.initView(section.questionsets[0].id);
+                    if (angular.isDefined(section.questionset)) {
+                        service.initView(section.questionsets[0].id);
+                    } else {
+                        // jump to first questionset of the section in breadcrumb
+                        let section_from_service = service.project.catalog.sections.find(x => x.id === section.id)
```

Comment:
Need an angularJS version of `.find()` here?

Reply:
You can use:

```
var section_from_service = $filter('filter')(service.project.catalog.sections, {
    id: section.id
})[0]
```

Reply:
this works, thanks!